src/test/kotlin/com/workos/test/organizations/OrganizationsApiTest.kt

```diff
@@ -1,12 +1,15 @@
 package com.workos.test.organizations
 
 import com.github.tomakehurst.wiremock.client.WireMock.* // ktlint-disable no-wildcard-imports
+import com.workos.auditlogs.AuditLogsApi
 import com.workos.common.exceptions.UnauthorizedException
 import com.workos.organizations.OrganizationsApi
 import com.workos.organizations.OrganizationsApi.CreateOrganizationOptions
+import com.workos.organizations.OrganizationsApi.CreateOrganizationRequestOptions
 import com.workos.organizations.OrganizationsApi.UpdateOrganizationOptions
 import com.workos.test.TestBase
 import org.junit.jupiter.api.Assertions.assertThrows
+import java.util.*
```

Comment:
```suggestion
```

Reply:
I don't think this import is used, but `./gradlew ktlintFormat` won't remove a wildcard import automatically.

Reply:
I just checked and it looks like it's not currently used.  Just removed it from the import list for the test and pushed up the change.